tests/PySys/local_mqtt/local_mqtt_pub_sub_retain/run.py

```diff
@@ -0,0 +1,78 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate local publishing with retain flag and subscribing:
+
+Given a configured system
+When we publish a message with retain flag to temperature sensor alaram topic (simulate set alaram)
+When we publish an empty message with retain flag to pressure sensor alarm topic (simulate clear alarm)
+When we start tedge sub in the background to subscribe to tempearture sensor alarm topic
+When we start tedge sub in the background to subscribe to pressure sensor alarm topic
+Then we kill both the subscribers
+Then we find the retained message in the output of temperature subscriber log
+And no message in the pressure sensor subscriber log
+"""
+
+
+class MqttPublishWithRetain(BaseTest):
+    def setup(self):
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
+
+        self.addCleanupFunction(self.retain_cleanup)
+
+    def execute(self):   
+
+        pub_to_set_alarm = self.startProcess(
+            command=self.sudo,
```

Comment:
`sudo` should not be required here.

Reply:
updated tests to use tedge MQTT pub/sub as a normal user.